docs/docsite/rst/community/documentation_contributions.rst

```diff
@@ -109,8 +109,8 @@ Drop the ``--user`` option in the following commands if you use a virtual enviro
 
    .. code-block:: bash
 
-    pip install --user -r docs/docsite/requirements.txt # This file installs the unpinned dependency versions.
-    pip install --user -r tests/requirements.txt # This file installs tested dependency versions.
+    pip install --user -r docs/docsite/requirements.txt # Installs the unpinned dependency versions.
```

Comment:
Why would we want unpinned deps on contributor machines? Don't they deserve stability too?

Reply:
Yeah but they get the freedom to choose. If they are savvy enough maybe they could do some testing of the deps.

In all honesty though this has been in the docs longer than I've been around the project so I'm assuming there is probably some historical reason for this. Maybe @samccann or @felixfontein could enlighten us?

Either way you've got a point and a bit of extra doc, even some kind of warning about potential instability, wouldn't hurt.

Reply:
I guess that's fair. Though, it wouldn't hurt to actually have a constraints file in repo for this (or both, combined? — I haven't checked how similar they are).

Reply:
Thanks again. I've flipped the order of the two commands in the docs so that the user is likely to see the one with pinned dependencies and choose that option.

It probably wouldn't hurt to have a constraints file in this repo but I would honestly prefer to handle that in a separate issue.

Reply:
Totally, I wasn't implying that everything should be bundled in this PR. I prefer atomic PRs myself :)